R/Archiver.R

```diff
@@ -6,6 +6,8 @@ Archiver <- R6::R6Class( # nolint: object_name_linter.
     #' @description Returns an `Archiver` object.
     #'
     #' @return an `Archiver` object
+    #' @examples
+    #' archiver <- getFromNamespace("Archiver", "teal.reporter")$new()
```

Comment:
Please import the function/class and use imported.
```suggestion
    #' Archiver <- getFromNamespace("Archiver", "teal.reporter")
    #' Archiver$new()
```

Reply:
alrighty, that will take some time but is actually more visible for the end user on what's goin on : )
Will do one task first in here https://github.com/insightsengineering/teal.data/issues/238 and then get back to this one

Reply:
Changed this in 5d000820789beec4a2359841f148f8b9b3e0bb2b for other examples as well